main.go

```diff
@@ -53,6 +53,14 @@ func main() {
 	if err != nil {
 		log.WithField("Error", err).Fatal("Could not fetch config options for etcd.")
 	}
+	oldCfg, err := etcdmgmt.ReadEtcdConfig()
```

Comment:
Oh no. I think you missed the point. I was avoiding code duplication. With whatever you're suggesting, I'll have to be calling `GetNewEtcdConfig` inside `ReadEtcdConfig` and then again in main().

I think it'll be better off if I just rename `GetNewEtcdConfig` to `GetEtcdConfig`. And let the function figure out whether to read it from file, or create new ?